news/128.2.new.rst

```diff
@@ -0,0 +1 @@
+Add hooks for ``thinc`` and ``thinc.banckends.numpy_ops`` which contain data files and hidden imports. These hooks were created to make spacy work correctly.
```

Comment:
As a final nitpick, perhaps put spacy in the backtick-quotes so it's rendered properly. Or perhaps just drop the last sentence, as those hooks stand on their own.